.github/workflows/build.yml

```diff
@@ -78,14 +78,15 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           files: /tmp/test-coverage.txt
-          fail_ci_if_error: true
+          fail_ci_if_error: false
```

Comment:
this seems to be broken at the moment, so just ignoring failure for now

Reply:
Damn, that's much smarter than my solution of re-running it loads 🥴